src/Device/Driver/XCVario.cpp

```diff
@@ -144,6 +153,35 @@ PXCV(NMEAInputLine &line, NMEAInfo &info)
   return true;
 }
 
+bool
+XVCDevice::XCV(NMEAInputLine &line, NMEAInfo &info)
+{
+  double value;
+  char topic[16];
+  line.Read(topic, 16);
+  if( StringIsEqual( topic, "crew-weight") ){
```

Comment:
Bad coding style

Reply:
stray spaces removed and repetition of constant